pandas/io/formats/format.py

```diff
@@ -1956,7 +1956,7 @@ def __call__(self, num: int | float) -> str:
         """
         Formats a number in engineering notation, appending a letter
         representing the power of 1000 of the original number. Some examples:
-
+        >>> format_eng = EngFormatter()
         >>> format_eng(0)       # for self.accuracy = 0
```

Comment:
The problem seems to be on these comments. You are creating an instance of `EngFormatter` with default parameters, but the comments are saying that accuracy should be `0`, and other paramter values below.

This seems decently tested in `tests/io/formats/test_eng_formatting.py`, so I don't think there should be an obvious bug. Can we start by creating an instance of `EngFormatter` for each case, with the parameters in the comments (and you can remove the comments, since it'll be already in the code).

Does this make sense?

Reply:
Yes it does thank you!